src/Todo.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Todo({ id, content, onClick }) {
```

Comment:
```suggestion
function Todo({ id, content, onClickDone }) {
```

여기도 onClickDone으로 하는 게 좋습니다.